apify-api/scripts/add-redoc-legacy-urls-to-tags.py

```diff
@@ -0,0 +1,48 @@
+import ruamel.yaml
+import csv
+import json
+from slugify import slugify
+from ruamel.yaml.scalarstring import LiteralScalarString
+import re
+from functools import reduce
+
+yaml = ruamel.yaml.YAML(typ='rt')
+yaml.indent(mapping=2, sequence=2, offset=0)
+
+# https://stackoverflow.com/questions/78075152/preserve-line-wrapping-in-ruamel-yaml
+# needed to convert flow sclars (>-) to literal blocks (|)
+#yaml.width = 10000
+
+# https://stackoverflow.com/questions/42094599/preserving-quotes-in-ruamel-yaml
+yaml.preserve_quotes = True
+
+# non empty null
+# https://stackoverflow.com/questions/57172997/yaml-dumper-outputs-blank-instead-of-null-from-none-value-how-to-make-it-output
+#def my_represent_none(self, data):
+#    return self.represent_scalar(u'tag:yaml.org,2002:null', u'null')
+#
+#yaml.representer.add_representer(type(None), my_represent_none)
+
+
+# Load tags
+tags_file_path = '../openapi/components/tags.yaml'
+with open(tags_file_path, 'r') as file:
+    tags = yaml.load(file)
+    for tag in tags:
+      print(tag)
+      safe_tag = re.sub(r'\/', "", tag['name'])
```

Comment:
not sure how this works in python, i would expect a global modifier, since there could be multiple slashes. but looking at our tags, we probably always have only one (or none), so i guess its fine this way too

Reply:
It's a one-off script. It worked for this use case. I'm leaving the scripts behind to document my thought process and have a paper trail of the patterns for manipulating the yaml for future reference.